Makefile

```diff
@@ -158,32 +158,26 @@ local-unit-test: local-unit-test-backend local-unit-test-processing # Run all ba
 local-unit-test-backend: # Run container-unittest target in `backend` Docker container.  If path arg provided, just run those specific backend tests
 	@if [ -z "$(path)" ]; then \
 	    echo "Running all backend unit tests"; \
-		export CI=""; \
-		ci_env=""; \
-		if [ ! -z "$(CODECOV_TOKEN)" ]; then \
-			ci_env=$$(bash <(curl -s https://codecov.io/env)); \
-			CI=true; \
-		fi; \
-	    docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -e CI $$ci_env -T backend \
-	    bash -c "cd /single-cell-data-portal && make container-unittest && if [ \"${CI}\" == "true" ]; then apt-get update && apt-get install -y git && bash <(curl -s https://codecov.io/bash) -cF backend,python,unitTest; fi"; \
+	    docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -T backend \
+	    bash -c "cd /single-cell-data-portal && make container-unittest;" \
 	else \
 		echo "Running specified backend unit test(s): $(path)"; \
 		docker-compose $(COMPOSE_OPTS) run --rm -e DEV_MODE_COOKIES= -T backend \
 		bash -c "cd /single-cell-data-portal && python -m unittest $(path)"; \
 	fi
 
+.PHONY: all-local-unit-test-backend
+all-local-unit-test-backend: # Run container-unittest target in `backend` Docker container.  If path arg provided, just run those specific backend tests
```

Comment:
This was necessary because there already is a `local-unit-test-backend` with an if-else clause that broke the tests for some reason. (the `else` clause was getting executed after the `if` clause and idk why).